CHANGELOG.md

```diff
@@ -10,7 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
    Unreleased section, uncommenting the header as necessary.
 -->
 
-<!-- ## Unreleased -->
+## Unreleased
+
+* lit-extended: Attribute bindings with a `?` suffix on the name now act as boolean
+  attributes. The attribute will be removed for falsey values and set to `''` for
+  truthy values.
```

Comment:
Maybe make it clear that this follows the HTML behavior:

From the HTML standard:

The presence of a boolean attribute on an element represents the true value, and the absence of the attribute represents the false value.

If the attribute is present, its value must either be the empty string or a value that is an ASCII case-insensitive match for the attribute's canonical name, with no leading or trailing whitespace.